presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/README.md

```diff
@@ -0,0 +1,148 @@
+# Presto planner and optimizer tests
+
+Testing planner and optimizer functionality generally requires generating a
+plan and then validating it against a set of expectations. The challenge is
+expressing the expectations in a way that's concise and resilient to changes in
+the objects Presto uses to represent the plan and the code that generates the
+plan.
+
+Unfortunately, that list includes the following:
```

Comment:
Suggest changing this to:
The expectations include the following:
(Or is it "objects" or "code" instead of "expectations"?